Assets/MixedRealityToolkit.Services/InputSystem/Utilities/CanvasUtility.cs

```diff
@@ -61,8 +78,8 @@ private void Start()
             {
                 if (canvas.worldCamera == null)
                 {
-                    Debug.Assert(MixedRealityToolkit.InputSystem.FocusProvider.UIRaycastCamera != null, this);
-                    canvas.worldCamera = MixedRealityToolkit.InputSystem.FocusProvider.UIRaycastCamera;
+                    Debug.Assert(InputSystem?.FocusProvider?.UIRaycastCamera != null, this);
```

Comment:
>? [](start = 59, length = 1)

Same question in other places (i.e. why does this add new null checks against FocusProvider?)

Reply:
to protect against null-reference exceptions if the FP is ever not set

Reply:
i am fine removing them. it just felt wrong to not have them when the code is dereferencing FocusProvider

Reply:
in retrospect, i would like to keep these checks to allow us to avoid expensive nullref exceptions.